apps/infra/api/routes/etl.py

```
@@ -0,0 +1,226 @@
+class SyncStatusResponse(BaseModel):
+    status: str
+    message: str
+    task_id: Optional[str] = None
+    address: str
+    chain_id: int
+    mode: str
+    transactions_count: Optional[int] = None
+
+
+# Background task to sync transactions
+async def sync_transactions_task(
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)